site/content/en/docs/Guides/Client SDKs/csharp.md

```diff
@@ -85,11 +112,62 @@ This process is non-blocking internally.
 agonesSDK.WatchGameServer((gameServer) => { Console.WriteLine($"Server - Watch {gameServer}");});
 ```
 
+### Alpha: PlayerConnect
+
+This method increases the SDK’s stored player count by one, and appends this playerID to GameServer.Status.Players.IDs.
+Returns true and adds the playerID to the list of playerIDs if the playerIDs was not already in the list of connected playerIDs.
+
+```csharp
+bool ok = PlayerConnectAsync(playerId);

```

Comment:
Thank you for writing the documentation!
I suggest the same as calling the actual method.
```suggestion
bool ok = await agones.Alpha().PlayerConnectAsync(playerId);
```